userProfile/privateProfile.mjs

```diff
@@ -20,6 +20,18 @@ router.route("/profile").get(auth0Middleware(), async (req, res) => {
   }
 }).all((req, res)=>    respondWithError(res, 405, "Improper request method. Use GET instead"))
 
+router.route("/profile/:id/update").put(auth0Middleware(), async (req, res) => {
```

Comment:
we don't need `:id/` here because we know who it is.

Reply:
we can also discard the `update` because this can just be the PUT method (tuck it in above before the `.all`)